battleship.py

```diff
@@ -62,8 +62,14 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
-
+    arr=[]

```

Comment:
@Vedvundmsit, you can change the arr variable name to any other meaningful variable to make the code readable.